src/common/components/repository-row/dropdowns/template-yaml.js

```diff
@@ -1,12 +1,12 @@
-export default function templateYaml(repoName, storeName) {
+export default function templateYaml(storeName) {
   let name;
 
   // If the snap isn't registed on the store change the first template line
   if (!storeName) {
     name = `# After registering a name on build.snapcraft.io, commit an uncommented line:
-  # name: ${repoName}`;
+  # name: registered-name`;
```

Comment:
[Spec](https://docs.google.com/document/d/1koTwI3cRr2S2vj5zHKKbR7aWxLRfShDVq2gHQ29cDyU/edit#heading=h.30j8f330tbd6) says it should be `name: your-registered-name`